lucien-ios/LoginScreen/RootViewController.swift

```diff
@@ -58,7 +58,7 @@ final class RootViewController: UIViewController, GIDSignInDelegate, GIDSignInUI
         guard let idToken = user.authentication.idToken else { return }
         loginViewModel.authenticateUser(code: idToken) {
             self.loginViewModel.hasCollection { result in
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Unused Closure Parameter Violation: Unused parameter "result" in a closure should be replaced with _. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Aunused_closure_parameter)